packages/adapter-auto/index.js

```diff
@@ -84,7 +84,7 @@ async function get_adapter() {
 
 				console.log(`Successfully installed ${match.module}.`);
 				console.warn(
-					`\nIf you plan on staying on this deployment platform, consider replacing @sveltejs/adapter-auto with ${match.module}. This will give you faster and more robust installs, and more control over deployment configuration.\n`
+					`\nIf you plan on staying on this deployment platform, consider replacing @sveltejs/adapter-auto with ${match.module}. This will give you faster, more robust installs, and more control over deployment configuration.\n`
```

Comment:
it reads a bit weird to me like "faster", "more robust install", and "more control" as opposed to "faster, more robust installs", and "more control". How about we just side-step the issue by simplifying what we're saying? I'm not sure what difference in robustness there is
```suggestion
					`\nIf you plan on staying on this deployment platform, consider replacing @sveltejs/adapter-auto with ${match.module}. This will give you faster installs and more control over deployment configuration.\n`
```